### PR TITLE
owners: update community plugin maintainers

### PR DESCRIPTION
**Pull request by Rugvip**
A couple of updates to the community plugin maintainers:

- Reordered to be alphabetical
- Removed @tudi2d, who is now at @lmu
- Added @nickboldt

Please approve if these are all fine @backstage/community-plugins-maintainers

### CODE REVIEW

**Review by tudi2d — APPROVED** [review]

File: OWNERS.md [hunk 1]
```diff
@@ -114,10 +114,10 @@ Scope: Tooling and Community Repo Maintainers for the Backstage [Community Plugi
 
 | Name                 | Organization | GitHub                                      | Discord      |
 | -------------------- | ------------ | ------------------------------------------- | ------------ |
+| André Wanlin         | Spotify      | [awanlin](https://github.com/awanlin)       | `ahhhndre`   |
 | Bethany Griggs       | Red Hat      | [BethGriggs](https://github.com/BethGriggs) | `bethgriggs` |
+| Nick Boldt           | Red Hat      | [nickboldt](https://github.com/nickboldt)   | `nboldt`     |
 | Tomas Kral           | Red Hat      | [kadel](https://github.com/kadel)           | `tomkral`    |
```
[the remark this answers is not in the document]
Comment (Rugvip) [reply OWNERS.md:120]: Was about to suggest this too but wanted to keep it simple :sweat_smile:

**Review by vinzscam — APPROVED** [review]

**Comment by Rugvip** [comment]
@nickboldt could you please approve if you are fine with this as well?

**Comment by nickboldt** [comment]
Can't approve with the UI as I'm not already a maintainer... but +1

/approve

**Comment by Rugvip** [comment]
@nickboldt any form of approval is alright, thank you :grin:

**Pull request merged**